init-tools.cmd

```diff
@@ -108,9 +107,6 @@ if not [%RESTORE_PORTABLETARGETS_ERROR_LEVEL%]==[0] (
   exit /b %RESTORE_PORTABLETARGETS_ERROR_LEVEL%
 )
 
-:: Copy Roslyn Compilers Over to ToolRuntime
-Robocopy "%PACKAGES_DIR%\Microsoft.Net.Compilers\%ROSLYNCOMPILERS_VERSION%\." "%TOOLRUNTIME_DIR%\net46\roslyn\." /E
```

Comment:
This isn't used anywhere and was already removed from the `build-tools` copy. New code is favoring direct references to the nuget package contents.